.nengobones.yml

```diff
@@ -12,26 +12,27 @@ license_rst: {}
 
 manifest_in:
   global-exclude:
-    - "*.swp"  # exclude VIM swp files
+    - "*.swp" # exclude VIM swp files
 
 travis_yml:
-  python: 3.7
+  python: 3.9
```

Comment:
We don't officially support Python 3.9 yet in Nengo, though that's not to say it might not all work fine.

Reply:
Unofficially, I'm currently using Python 3.9, but with 3.8 the issue was Windows specific so who knows.

Reply:
I think it doesn't hurt to run the tests on 3.9 too as long as they pass. I also think that the "Python 3.6+" formulation in the the readme is fine (NengoSPA is more platform independent than Nengo, so it is even more unlikely that a new Python release breaks things). Might consider taking the classifier out of the setup.py ... but that I would have to re-add it later.